### PR TITLE
configuring aws codepipeline with ecs and alb

### PR DESCRIPTION
**Pull request by robmcohn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ The preview server is a headless chromium instance that generates screenshots an
 Run https://github.com/webaverse/app, then:
 
 https://127.0.0.1:3001/screenshot.html?url=https://webaverse.github.io/assets/male.vrm&ext=vrm&type=png
+
```
Comment (webaverse-storage) [inline README.md]: Probably no reason for this in the source

File: Dockerfile [hunk 2]
```diff
@@ -1,21 +1,27 @@
-FROM buildkite/puppeteer:latest
+# Use node 4.4.5 LTS
```
Comment (avaer) [inline Dockerfile]: Is this accurate? This would be a very old version of node…

**Review by avaer — COMMENTED** [review]
Some small cleanups but overall it looks good.

File: Dockerfile (hunk 2 above)
Comment (robmcohn) [reply Dockerfile]: It's accurate. It was the simplest, most efficient way I could make it work. And it's not that old a version; it's using v14: https://github.com/buildkite/docker-puppeteer/blob/master/Dockerfile

Comment (avaer) [reply Dockerfile]: Oh so it's not node 4.5, but 14.

Comment (robmcohn) [reply Dockerfile]: That node line is commented out. I think I tried it for a minute, but ended up going back to using buildkite/puppeteer, which is based on node14

**Review by avaer — APPROVED** [review]

**Pull request merged**